lnbits/wallets/alby.py

```diff
@@ -47,7 +47,8 @@ async def status(self) -> StatusResponse:
         data = r.json()["balance"]
         if r.is_error:
             return StatusResponse(data["error"], 0)
```

Comment:
Unrelated to the PR, but this code looks really weird.

1. First, we parse the JSON, but we check if there is an error only afterwards.
2. What's even weirder is that we access `data["error"]`, but I don't think `data` is a dict at this point.

Maybe this section should be this instead?

I checked the [documentation here](https://guides.getalby.com/developer-guide/v/alby-wallet-api/reference/api-reference/account) but I haven't tested it. 

```suggestion
        if r.is_error:
            error_message = r.json()["message"]
            return StatusResponse(error_message, 0)
        data = r.json()["balance"]
```

Reply:
you are correct, it should message, and consistent with other methods in same class. 

Should we have unit tests to simulate this testing? Most projects at this size start to have them.

Reply:
Same pattern appears in opennode.py

https://github.com/lnbits/lnbits/blob/e1bb2113ed92556c6227f6de14126570f808007a/lnbits/wallets/opennode.py#L59-L63

Reply:
I reviewed the other wallet files (e.g. lnpay, etc). Every single one has a slightly different style. Not sure what you guys want to do for handling errors and exceptions, so I originally templated the alby wallet integration off of opennode.py

Reply:
> Same pattern appears in opennode.py

Can you please fix opennode in this PR the same way like you fixed alby?